rest-api/config/config_prod.json

```diff
@@ -17,7 +17,11 @@
     },
     "exporter@all-of-us-rdr-prod.iam.gserviceaccount.com": {
       "roles": ["exporter"]
-    }
+    },
+    "awardee-pitt@all-of-us-rdr-prod.iam.gserviceaccount.com": {
```

Comment:
We may want to hold off on making the config changes in prod until the service accounts have been created. I'm not sure which awardees will be going first.

Reply:
removed